frontend/src/translations/ru.global.json

```
@@ -0,0 +1,551 @@
+          "title": "Последнее обновление телеметрии"
+        },
+        "node_agent_name": {
+          "text": "Протокол NEAR может иметь несколько реализаций, поэтому агент — это имя этой реализации, где «near-rs» - это ${agent_name_url}.",
+          "url_text": "официальная реализация",
+          "title": "Имя агента узла"
+        },
+        "node_agent_version_or_build": {
+          "title": "Версия агента узла / Построение"
+        }
+      },
+      "ValidatorMetadataRow": {
+        "pool_info": {
+          "website": "Сайт",
+          "email": "Email",
+          "twitter": "Twitter",
+          "discord": "Discord",
+          "description": "Описание"
+        },
+        "pool_info_tip": {
+          "text": "Если вы владелец узла, не стесняйтесь заполнить ${pool_info_url}, чтобы продвигать собственный узел!",
+          "url_text": "информацию о себе"
+        }
+      },
+      "NodesEpoch": {
+        "current_epoch_start": "Начало текущей эпохи",
+        "block": "Блок",
+        "complete": "завершено",
+        "remaining": "осталось"
+      },
+      "NodesContentHeader": {
+        "nodes": "Узлы"
+      },
+      "NodeNav": {
+        "validating": "Подтверждение",
+        "proposed": "Предложено"
+      },
+      "NodesCard": {
+        "nodes_validating": "Валидаторы",
+        "total_supply": "Общее предложение",
+        "total_stake": "Всего в стейкинге",
+        "seat_price": "Стоимость места валидатора"
+      },
+      "Nodes": {
+        "validator": "Валидатор"
+      },
+      "NodeRow": {
+        "online": {
+          "title": "В сети",
+          "text": "Узлов в сети"
+        }
+      }
+    },
+    "transactions": {
+      "Transactions": {
+        "no_transactions": "Нет транзакций"
+      },
+      "ActionMessage": {
+        "CreateAccount": {
+          "new_account_created": "Создан новый аккаунт: "
+        },
+        "DeleteAccount": {
+          "delete_account": "Удалить аккаунт ",
+          "and_transfer_fund_to": " и перевести оставшиеся средства на "
+        },
+        "DeployContract": {
+          "contract_deployed": "Развёрнутый контракт: "
+        },
+        "FunctionCall": {
+          "called_method": "Вызываемый метод: '${method_name}' в контракте: ",
+          "arguments": "Параметры:"
+        },
+        "Transfer": {
+          "transferred": "Переведено ",
+          "to": "  "
+        },
+        "Stake": {
+          "staked": "В стейкинге: ",
+          "with_key": "с использованием публичного ключа ${public_key}..."
+        },
+        "AddKey": {
+          "new_key_added": "Добавлен новый ключ для ",
+          "with_permission_and_nounce": "с разрешением ${permission} и одноразовым номером ${nonce}",
+          "access_key_added_for_contract": "Ключ доступа добавлен для контракта ",
+          "with_permission_call_method_and_nounce": "с разрешением для вызова ${methods} методов с одноразовым номером ${nonce}",
```

```suggestion
          "with_permission_call_method_and_nounce": "с разрешением для вызова ${methods} методов",
```
The same as on [246 line](https://github.com/near/near-explorer/pull/738/files?file-filters%5B%5D=.json#diff-bd99d612ee388041e267198e6b6b9f000fb4da08d04e85e27d8b487f6d7c92b2R246). @frol Am I right?